modules/nw-route-specific-annotations.adoc

```diff
@@ -68,7 +68,8 @@ This value is applicable to re-encrypt and edge routes only. For more informatio
 |===
 [.small]
 --
-1. If the number of IP addresses and CIDR ranges in an allowlist exceeds 61, they are written into a separate file that is then referenced from `haproxy.config`. This file is stored in the `var/lib/haproxy/router/allowlists` folder.
+1. By default, the router reloads every 5s which resets the balancing connection across pods from the beginning. This means that the `roundrobin` state is not preserved across reloads. This algorithm works best when pods have roughly identical computing capabilites and storage capacity. If your application or service has continuously changing endpoints, for example, due to using a CI/CD pipeline, this can result in uneven balancing. In this case, it is recommended to use a different algorithm.
+2. If the number of IP addresses and CIDR ranges in an allowlist exceeds 61, they are written into a separate file that is then referenced from `haproxy.config`. This file is stored in the `var/lib/haproxy/router/allowlists` folder.
 +
```

Comment:
Better change `var/lib/haproxy/router/allowlists` to `/var/lib/haproxy/router/allowlists`  (missing the "/" character), others look good to me, thanks.